deps.qt/qt6.ps1

```diff
@@ -143,6 +143,7 @@ function Configure {
     $CMakeTarget = @{
         x64 = 'x64'
         x86 = 'Win32'
+        arm64 = 'arm64'
     }
 
     $Options = ($Options -join ' ') -replace '-G Visual Studio \d+ \d+','-G Ninja' -replace "-A $($CMakeTarget[$Target])",''
```

Comment:
Qt requires a working Qt host with precompiled tools (moc, uic, etc.). Here's a quick and dirty hack that got me working locally. We would probably have to have the arm64 job wait for a successful x64 run, download the build, extract it somewhere, and point the arm64 job at that Qt directory.
```diff
diff --git a/deps.qt/qt6.ps1 b/deps.qt/qt6.ps1
index a7aea57..f9ee2a9 100644
--- a/deps.qt/qt6.ps1
+++ b/deps.qt/qt6.ps1
@@ -146,6 +146,13 @@ function Configure {
         arm64 = 'arm64'
     }

+    if ( $Target -eq "arm64" ) {
+        $Options += @(
+            '-DQT_QMAKE_TARGET_MKSPEC:STRING=win32-arm64-msvc'
+            '-DQT_HOST_PATH:PATH=C:\Qt\6.6.1\msvc2019_64'
+        )
+    }
+
```